go-controller/pkg/node/node.go

```diff
@@ -768,11 +768,7 @@ func doesEPSliceContainEndpoint(epSlice *discovery.EndpointSlice,
 }
 
 func configureSvcRouteViaBridge(bridge string) error {
-	gwIPs, _, err := getGatewayNextHops()
-	if err != nil {
-		return fmt.Errorf("unable to get the gateway next hops, error: %v", err)
-	}
-	return configureSvcRouteViaInterface(bridge, gwIPs)
+	return configureSvcRouteViaInterface(bridge, DummyNextHopIPs())
```

Comment:
Shall it try to get gateway nexthops, then use the dummy one if default route doesn't exist?

Reply:
@jcaamano ack+, I noticed the asymmetry between host and GW, thus the asking. I saw you had another comment covering this. Thanks!

Reply:
Resolving this conversation; thanks for providing an accurate reply @jcaamano .